tests/qchem/test_tapering.py

```diff
@@ -989,3 +989,44 @@ def test_split_pauli_sentence(ps_size, max_size):
         split_sentence = {**split_sentence, **ps}
 
     assert sentence == qml.pauli.PauliSentence(split_sentence)
+
+
+@pytest.mark.jax
+@pytest.mark.parametrize(
+    ("symbols", "geometry", "charge"),
+    [
+        (
+            ["H", "H"],
+            np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.40104295]], requires_grad=True),
+            0,
+        ),
+        (
+            ["He", "H"],
+            np.array(
+                [[0.0, 0.0, 0.0], [0.0, 0.0, 1.4588684632]],
+                requires_grad=True,
+            ),
+            1,
+        ),
+    ],
+)
+def test_taper_jax_jit(symbols, geometry, charge):
+    r"""Test that an observable can be tapred within a jax-jit workflow."""
```

Comment:
How this test checks for wire ordering issues?

Reply:
Added a test case that actually checks it.